### PR TITLE
MTR auth

### PR DESCRIPTION
**Pull request by sohailta**
Updates  per recommendations from AAD Team.

### CODE REVIEW

File: Teams/rooms/rooms-authentication.md [hunk 1]
```diff
@@ -83,7 +85,7 @@ However, because Microsoft Teams Rooms uses [resource owner password credentials
 - You must have Exchange Server 2016 CU8 or later, or Exchange Server 2019 CU1 or later.
 - You must have Skype for Business Server 2015 CU5 or later, or Skype for Business Server 2019 or later.
 - MFA isn't supported regardless of the topology you have.
-- If you use a third-party authentication provider that's supported by Azure AD, it must support OAuth and use  resource owner password credentials authorization.
+- If you use a third-party authentication provider that's supported by Azure AD, it must support OAuth 2.0 and active authentication flows throgh WS-Trust or WS-Federation.
```
Comment (Copowell) [inline Teams/rooms/rooms-authentication.md:88]: I would reword this to "If you use a third-party authentication provider that's supported by Azure AD, it must support an active authentication flow through WS-Trust." The third-party auth provider doesn't have to support OAuth2, just be properly federated in Azure AD and support WS-Trust.

File: Teams/rooms/rooms-authentication.md [hunk 2]
```diff
@@ -62,17 +62,19 @@ Before you begin, make sure you understand the identity models to use with Offic
...
 
 ## Hybrid modern authentication
 
-To ensure successful authentication to your on-premises Exchange server and/or Skype for Business server, you must make sure the resource account that's used with Microsoft Teams Rooms is configured to get authorization from Azure AD. To learn more about hybrid identity and methods that work for your organization, read the following topics: 
+To ensure successful authentication to your on-premises Exchange server and/or Skype for Business server, you must make sure the resource account that's used with Microsoft Teams Rooms is configured to get authorization from Azure AD. 
+
+Teams Rooms authentication flows vary depending on your authenitcation configuration. For customers using managed domain, Teams Rooms uses [OAuth 2.0 Resouece Owner Password Credentials](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) with Azure Active Diretory, however for customers using federated domain, [OAuth 2.0 SAML bearer assertion flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-saml-bearer-assertion) is used.
```
Comment (Copowell) [inline Teams/rooms/rooms-authentication.md:73]: I would reword it this way instead:

"Teams Rooms authentication flows vary depending on your authentication configuration. For customers using a managed domain, Teams Rooms uses [OAuth 2.0 Resource Owner Password Credentials](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) with Azure Active Directory, however for customers using a federated domain, [OAuth 2.0 SAML Bearer Assertion Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-saml-bearer-assertion) is used."

Comment (JefTek) [reply Teams/rooms/rooms-authentication.md]: Minor miss-spelling for "authentication" in 2nd paragraph.   Perhaps also say "For customers managed using authentication for their domain in Azure AD, Teams Room uses.."      then have "For customers using federated authentication for their domain in Azure AD, Teams Rooms uses...."

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-authentication.md [hunk 3]
```diff
@@ -26,9 +26,9 @@ Modern authentication is supported on Microsoft Teams Rooms version 4.4.25.0 and
...
+When you use modern authentication with the Microsoft Teams Rooms application, Active Directory Authentication Library (ADAL) is used to connect to Microsoft Teams, Exchange, and Skype for Business. A Microsoft Teams Rooms device is a shared device and performs a nightly reboot to ensure smooth functioning and to get critical operating system, driver, firmware, or application updates. The modern authentication mechanism uses the [resource owner password credentials](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) authorization grant type in OAuth 2.0, which doesn't require any user intervention. This is one of the key differences between how modern authentication works for user accounts versus resource accounts that are used by the Microsoft Teams Rooms application. Because of this, Microsoft Teams Rooms resource accounts shouldn't be configured to use multi-factor authentication (MFA), smart card authentication, or client certificate-based authentication (which are all available for end users).
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:29]:
```suggestion
When you use modern authentication with the Microsoft Teams Rooms application, Active Directory Authentication Library (ADAL) is used to connect to Microsoft Teams, Exchange, and Skype for Business. A Microsoft Teams Rooms device is a shared device and performs a nightly reboot to ensure smooth functioning and to get critical operating system, driver, firmware, or application updates. The modern authentication mechanism uses the [resource owner password credentials](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth-ropc) authorization grant type in OAuth 2.0, which doesn't require any user intervention. This is one of the key differences between how modern authentication works for user accounts versus resource accounts that are used by the Microsoft Teams Rooms application. Because of this, Microsoft Teams Rooms resource accounts shouldn't be configured to use multi-factor authentication (MFA), smart card authentication, or client certificate-based authentication (which are all available for end users).
```

Comment (JohanFreelancer9) [reply Teams/rooms/rooms-authentication.md:29]: Please remove the locale (en-us) from the link.

File: Teams/rooms/rooms-authentication.md [hunk 4]
```diff
@@ -62,17 +62,19 @@ Before you begin, make sure you understand the identity models to use with Offic
...
 To turn on modern authentication for Exchange Online, see [Enable modern authentication in Exchange Online](https://docs.microsoft.com/exchange/clients-and-mobile-in-exchange-online/enable-or-disable-modern-authentication-in-exchange-online). If you use Skype for Business Online, you should also make sure that modern authentication is turned on for Skype for Business Online. To learn more, see [Skype for Business Online: Enable your tenant for modern authentication](https://aka.ms/SkypeModernAuth).
 
-We recommended that you do not remove basic authentication policies for Exchange Online or disable basic authentication for your tenant before you have validated that Microsoft Teams Rooms devices can sign in to Exchange Online and Teams or Skype for Business Online successfully when the modern authentication setting is on and that there are no devices that are still configured to use basic authentication.
+We recommended that you do not remove basic authentication policies for Exchange Online or disable basic authentication for your tenant until you have validated that Microsoft Teams Rooms devices can successfully sign in with Exchange Online, Teams and Skype for Business Online.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:65]:
```suggestion
We recommended that you do not remove basic authentication policies for Exchange Online or disable basic authentication for your tenant until you have validated that Microsoft Teams Rooms devices can successfully sign in with Exchange Online, Teams, and Skype for Business Online.
```

Comment (JohanFreelancer9) [reply Teams/rooms/rooms-authentication.md:65]: Please insert comma after "Teams".

File: Teams/rooms/rooms-authentication.md [hunk 5]
```diff
@@ -62,17 +62,19 @@ Before you begin, make sure you understand the identity models to use with Offic
...
 
 ## Hybrid modern authentication
 
-To ensure successful authentication to your on-premises Exchange server and/or Skype for Business server, you must make sure the resource account that's used with Microsoft Teams Rooms is configured to get authorization from Azure AD. To learn more about hybrid identity and methods that work for your organization, read the following topics: 
+To ensure successful authentication to your on-premises Exchange server and/or Skype for Business server, you must make sure the resource account that's used with Microsoft Teams Rooms is configured to get authorization from Azure AD. 
+
+Teams Rooms authentication flows vary depending on your authentication configuration. For customers using a managed domain, Teams Rooms uses [OAuth 2.0 Resouece Owner Password Credentials](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth-ropc) with Azure Active Directory, however for customers using a federated domain, [OAuth 2.0 SAML Bearer Assertion Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-saml-bearer-assertion) is used.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:73]:
```suggestion
Teams Rooms authentication flows vary depending on your authentication configuration. For customers using a managed domain, Teams Rooms uses [OAuth 2.0 Resource Owner Password Credentials](https://docs.microsoft.com/azure/active-directory/develop/v2-oauth-ropc) with Azure Active Directory. However, for customers using a federated domain, [OAuth 2.0 SAML Bearer Assertion Flow](https://docs.microsoft.com/azure/active-directory/develop/v2-saml-bearer-assertion) is used.
```

Comment (JohanFreelancer9) [reply Teams/rooms/rooms-authentication.md:73]: Please merge changes as indicated.

File: Teams/rooms/rooms-authentication.md [hunk 6]
```diff
@@ -62,17 +62,19 @@ Before you begin, make sure you understand the identity models to use with Offic
...
+
+> [!NOTE]
+> Your identity provider may need specific configurations or settings for integration with Azure Active Directory or Office 365. Contact your identity provider if you need help with configurating authentication with Teams Rooms.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-authentication.md:76]:
```suggestion
> Your identity provider may need specific configurations or settings for integration with Azure Active Directory or Office 365. Contact your identity provider if you need help with configuring authentication with Teams Rooms.
```

Comment (JohanFreelancer9) [reply Teams/rooms/rooms-authentication.md:76]: Change "configurating" to "configuring".
